icu4c/source/i18n/unicode/simplenumberformatter.h

```diff
@@ -78,24 +78,12 @@ class U_I18N_API SimpleNumber : public UMemory {
      */
     void roundTo(int32_t power, UNumberFormatRoundingMode roundingMode, UErrorCode& status);
 
-#ifndef U_HIDE_DRAFT_API
     /**
      * Sets the number of integer digits to the given amount, truncating if necessary.
      *
-     * @draft ICU 75
+     * @stable ICU 75
```

Comment:
We agreed to remove truncateStart() in ICU 76. I think we should in fact do so.
Of course, we should not promote this replacement to stable in 76.
Even if we pushed removing truncateStart() to 77, we should leave the promotion to the usual BRS task.

So please don't do the promotion in this PR.

Reply:
> Even if we pushed removing truncateStart() to 77, we should leave the promotion to the usual BRS task.

Ok that's reasonable